frescobaldi_app/inputdialog.py

```diff
@@ -92,12 +95,13 @@ def getColor(
     if color is None:
         color = _savedColor
     dlg = QColorDialog(color, parent)
-    options = QColorDialog.ColorDialogOptions()
+    options = []
```

Comment:
It seems that `QColorDialog.ColorDialogOptions` disappeared without replacement and or-ing the individual options is the way to go, but a double-check would be welcome. My familiarity with Qt and PyQt is really limited.

Reply:
Replaced with the `QColorDialog.ColorDialogOption(0)` construct.